### ISSUE TITLE
Lambda Access point not working

### ISSUE THREAD

**Issue by dopan** [iterate-ch/cyberduck#14673]
Is CyberDuck expected to work with AWS Lambda Access Point? 

AWS docs mentions that lambda access point should work the same as standard Access Point, which works fine in CyberDuck with following configuration:
server: `s3.amazonaws.com`
path: `access-point-alias`

For access point if I keep configuration from above (specifying `lambda-access-point` alias as path), I am getting `PermanentRedirect` error (logs attached). 
[PermanentRedirect.log](https://github.com/iterate-ch/cyberduck/files/11470005/PermanentRedirect.log)

After some investigation and playing around, I was able to list Lambda Access Point root folder 
with the following config:
server:  `lambda-access-point alias ` + .s3.eu-central-1.amazonaws.com
path: empty

....however no other action works with this approach...e.g.:
- downloading some file gives `BadRequest`
- going to subfolder gives `Specified bucket does not exists`
(obviously the config is not correct, but I haven't came up with any other approach that would work here)

Would be grateful for any hints,
Thanks!

**Comment by egummett-aws** [comment]
I don't believe this is a duplicate. Amazon S3 Lambda Access Points and Multi-Region Access Points are different
https://docs.aws.amazon.com/AmazonS3/latest/userguide/MultiRegionAccessPointRestrictions.html
https://docs.aws.amazon.com/AmazonS3/latest/userguide/olap-use.html
